lib/faraday/adapter/em_http.rb

```diff
@@ -281,6 +281,9 @@ def check_finished
     warn 'Warning: no such file to load -- openssl. ' \
       'Make sure it is installed if you want HTTPS support'
   else
-    require 'faraday/adapter/em_http_ssl_patch'
+    require 'em-http/version'
```

Comment:
👋 Hello, again!

Thanks for working on this issue!

One little detail about our _optional_ gem dependencies:

We need to avoid `require` of non-guaranteed-to-be-available gems in this context of the fix. In other words, this Faraday adapter file must be `require`:able without having the em-http gem available.

The `dependency` block is where are sure that we can `require` the `em-http` gem. When a Faraday user configures usage of the `:em_http` adapter, the `dependency` method is run.

Hope this helps, and that it's not discouraging to run into _details_! Thanks for helping out!

Reply:
Alright, I've now moved the `require 'em-http/version'` call to the dependency block in each of the adapter files